src/modals/GeolocationModal.jsx

```diff
@@ -51,21 +51,27 @@ const GeolocationModal = ({ status, onClose, onAccept }) => {
         'Allow Qwant Maps to access your position so we can better help you find your way…',
         'geolocation'
       ),
-      button: _('Ok', 'geolocation'),
+      link: {
+        label: _('How to access the geolocation services?', 'geolocation'),
+        url: getLocalizedUrl('helpGeolocation'),
+      },
       className: 'modal__maps__denied',
     },
     NOT_ACTIVATED: {
       title: _('Houston,<br/> we have a (geolocation) problem&nbsp;🛰️', 'geolocation'),
       text: _(
         "We can't access your position.<br/> Please check that your geolocation services are enabled."
       ),
-      button: _('Ok', 'geolocation'),
+      link: {
+        label: _('How to access the geolocation services?', 'geolocation'),
```

Comment:
Should we add a space before the `?` :  `How to access the geolocation services ?`

Reply:
There isn't any space before punctuation in English, apparently this is a typically French mistake: https://ell.stackexchange.com/a/4870 :laughing: